test/ibmq/test_ibmq_logger.py

```diff
@@ -0,0 +1,152 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Tests related to logger setup via ``setup_logger()``."""
+
+import os
+import logging
+from tempfile import NamedTemporaryFile
+from unittest import skipIf, mock
+
+from qiskit.providers.ibmq.utils.utils import setup_logger
+
+from ..ibmqtestcase import IBMQTestCase
+
+# Constants used by `TestLogger`.
+QISKIT_IBMQ_PROVIDER_LOG_LEVEL = 'QISKIT_IBMQ_PROVIDER_LOG_LEVEL'
+QISKIT_IBMQ_PROVIDER_LOG_FILE = 'QISKIT_IBMQ_PROVIDER_LOG_FILE'
```

Comment:
They're probably better to be defined in `qiskit.providers.ibmq.__init__` since they're mainline constants.

Reply:
Would it be a good idea to include these constants under the html docs for `qiskit.providers.ibmq` so users are more aware of them? Or, is their intention just for having a more easier/better access with in the provider modules?

Reply:
Attributes don't show up very well in the docs now, so I wouldn't bother.